src/database/index.ts

```diff
@@ -4,19 +4,27 @@ interface Reading {
 
 // This is a fake database which stores data in-memory while the process is running
 // Feel free to change the data structure to anything else you would like
-const database: Record<string, Reading> = {};
```

Comment:
This must have renamed when updating the file to index.ts, not intended! Naughty IDE!